docs/assignment2.md

```diff
@@ -1 +1,157 @@
+
+## Implementation									
+Author(s): Sam, Gemma
+
+Our strategy for moving from our UML models to the implementation code was fairly simple. We started by setting up a 
+minimally-implemented class for each of the classes that we planned to include through our diagrams. To be specific,
+this involved adding our constructors, fields, and simple methods - mainly accessors and mutators - to each class as an
+easy first step. Then we began more in-depth work on some of the more central classes, particularly those that are
+relevant to our first two features as planned in Assignment 1. For us, this meant our **Main** class, **Room**, 
+**Item**, **Setting**, **Action**, and **Player**. This notably left out **TimeLimit** and **Obstacle** as classes that
+we plan on implementing as part of Assignment 3. In addition, we began work on the first iterations of JSON files used
+for the implementation of many different types of games. As we continued developing our **Main** class and the JSON
+files it would interact with, we realized we had modeled more complexity than necessary to describe (and later
+implement) our planned features. In line with this realization, we chose to eliminate the **Objective** class (thus now
+notably absent from all of our diagrams) as it was entirely redundant and unnecessary. In general, we continued with
+this strategy of following our diagrammed plan and eliminating redundancies or unnecessary complexity until we had
+implemented all of our features planned for Assignment 2. Furthermore, after getting as far as having a working game, we
+tested the implementation of each significant new piece of code to ensure that we could catch any bugs and solve them
+before they became too obscure or difficult to find.
+
+There were several key solutions we applied when implementing our system. First and foremost, we decided that the best
+way to make our game configurable or adaptable for new stories was to have it read and interpret JSON files that
+included all of the setting-specific information, such as room, item, or obstacle names. Next, we established that the
+simplest and most efficient method of continually reading the player's commands was to use a while loop within the
+**Main** class during runtime - any sort of command parser class seemed to be unnecessarily complex as well as involve
+more code being executed during runtime, likely making the game run slightly slower. Other key solutions were largely
+GUI-related: while our game does take place entirely within the terminal as specified by the assignment, we use ANSI
+colors to make the output shown to the player during the game more clearly readable and easily understood.
+
+The main Java class needed for executing the system in the source code can be found at the directory location:
+software-design-vu/src/main/java/Main.java
+
```

Comment:
Ok, you are mainly describing the Main class and it's JSON/input reading functionality but nothing on the other classes at all.